logstash.go

```diff
@@ -74,6 +74,8 @@ func (a *LogstashAdapter) Stream(logstream chan *router.Message) {
 			Hostname: m.Container.Config.Hostname,
 		}
 
+		tags := GetContainerTags(m.Container.Config.Env)
```

Comment:
I hadn't thought of it no... 😐  I'm sure that logspout has some function that runs when new containers are added, maybe I can look there and try to read once and just reuse afterwards...